<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreCalcProgressDialog</class>
 <widget class="QDialog" name="PreCalcProgressDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processing...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/gui.qrc">
    <normaloff>:/pluginIcon</normaloff>:/pluginIcon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel the Precalculation (this will also send a &quot;kill&quot; signal to all processes)</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="ui_copyAction">
   <property name="icon">
    <iconset resource="../resources/gui.qrc">
     <normaloff>:/copyIcon</normaloff>:/copyIcon</iconset>
   </property>
   <property name="text">
    <string>copyAction</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/gui.qrc"/>
 </resources>
 <connections/>
</ui>
